lib/auth.jsx

```diff
@@ -20,7 +20,7 @@ export default React.createClass( {
 	render() {
 		const { isAuthenticated, authPending } = this.props;
 		const submitClasses = classNames( 'button', 'button-primary', {
-			'pending': authPending
+			pending: authPending
 		} );
 
```

Comment:
some syntactic sugar. Destructuring assignment lets's you assign an extracted value to a new variable name.

```
const { isAuthenticated, authPending: pending } = this.props;
const submitClasses = classNames( 'button', 'button-primary', { pending } );
```